test/terraform_aws_cloudtrail_alarms_test.go

```diff
@@ -0,0 +1,38 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+
+)
+
+func TestTerraformAwsCloudTrailAlarms(t *testing.T) {
+	awsRegion := "us-west-2"
+
+	t.Parallel()
+
+	tempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "../", ".")
+	alarm_sns_topic_arn := "arn:aws:cloudwatch:us-west-2:123456789012:alarm:myCloudWatchAlarm-CPUAlarm-UXMMZK36R55Z"
```

Comment:
I'm not sure where this sns topic come from, but it probably belongs in the `examples/resources` along